doc/sphinx/source/quickstart/installation.rst

```diff
@@ -214,3 +232,15 @@ confirm that no errors are reported:
 .. code-block:: bash
 
     python setup.py test
+
+.. note::
+   Using ``pip`` as installation tool is very useful since it installs in place
+   (ie inside the current source code directory, and it does not perform a copy to
+   the target install directory ``site-packages``, unlike ``python setup.py install``, which
+   acts in an equivalent manner to ``make``); it also checks for needed dependencies
```

Comment:
I would remove the reference to `make`, because that's confusing for people who are not familiar with it.

Reply:
that is actually useful for people that are not familiar with python but are with compiled/built to target code - this stemmed from a question @rswamina asked me, I think it explains it ok

Reply:
anyhoo, I rephrased it and took out `make` :beer: